core/arch/arch.h

```diff
@@ -140,7 +140,6 @@ mixed_mode_enabled(void)
 #    define R12_OFFSET ((MC_OFFS) + (offsetof(priv_mcontext_t, r12)))
 #    define R13_OFFSET ((MC_OFFS) + (offsetof(priv_mcontext_t, r13)))
 #    define R14_OFFSET ((MC_OFFS) + (offsetof(priv_mcontext_t, r14)))
-#    define XFLAGS_OFFSET ((MC_OFFS) + (offsetof(priv_mcontext_t, xflags)))
```

Comment:
The PR title says this is redundant: can you point to the other definition of this offset, maybe add it to the PR description too?

Reply:
It's here: https://github.com/DynamoRIO/dynamorio/pull/6253/files#diff-5930c3a3d189a4f789955c17c5131c7015843070785649c71b254c92f7ded6b2R187

How should I point to that line? A GitHub link like the above or some plain text line like `core/arch/arch.h:187`?

Reply:
I think the later is better, and PR description updated.